pkgs/development/python-modules/bcolz/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildPythonPackage, fetchPypi, numpy, mock, setuptools_scm, isPy27, isPy35 }:
+
+buildPythonPackage rec {
+    pname = "bcolz";
+    version = "1.1.2";
+    name = "${pname}-${version}";
+
+    disabled = !(isPy35 || isPy27);
```

Comment:
surely it works on other interpreter versions as well?

Reply:
You're right. Removed.